### PR TITLE
Gabrielalao/fence 1338 upgrade flutter sdk ios handler to use the new method

### PR DESCRIPTION
**Pull request by gabrielalao**
## Linear
[FENCE-1338](https://linear.app/radarlabs/issue/FENCE-1338/upgrade-flutter-sdk-ios-handler-to-use-the-new-method-channel)

### CODE REVIEW

**Comment by gabrielalao** [comment]
Can you review this PR? @tjulien

**Comment by tjulien** [comment]
@gabrielalao Have you been able to test this on an iOS device, both in the foreground and background for an extended period of time (30 mins), while moving a few 100 feet to trigger location updates?

**Comment by gabrielalao** [comment]
@tjulien it's working well

File: example/ios/Runner/AppDelegate.m [hunk 1]
```diff
@@ -7,7 +7,6 @@ @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application
     didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
   [GeneratedPluginRegistrant registerWithRegistry:self];
-  [Radar initializeWithPublishableKey:@"prj_test_pk_0000000000000000000000000000000000000000"];
```
Comment (tjulien) [inline example/ios/Runner/AppDelegate.m:10]: For consistency, should this also be removed from the Android example code here?

https://github.com/radarlabs/flutter-radar/blob/master/example/android/app/src/main/java/io/radar/example/MainApplication.java#L12

Is the idea that `example/lib/main.dart.initRadar()` already has the call to `Radar.initialize()`?

Comment (gabrielalao) [reply example/ios/Runner/AppDelegate.m:10]: I checked that too, but android needs to be initialized on the java side as well. @tjulien

Comment (tjulien) [reply example/ios/Runner/AppDelegate.m:10]: Interesting.  For consistency, should we leave it in iOS as well then, and remove it from example dart lib?

Comment (gabrielalao) [reply example/ios/Runner/AppDelegate.m]: That's so weird, the actual logic are both, but in android, we have to call native initialize. Btw I updated the example @tjulien

**Review by tjulien — APPROVED** [review]

**Pull request merged**